server/JsDbg.Gdb/GdbDebugger.cs

```diff
@@ -304,11 +304,11 @@ class GdbDebugger : IDebugger {
             // NOTE that quotes can also appear in filenames, but they will be \ escaped
 
             string pythonResult = await this.QueryDebuggerPython(String.Format("GetCallStack({0})", frameCount));
-            
 
             List<SStackFrame> result = new List<SStackFrame>();
 
-            int index = 1; // Skip initial '"' character

```

Comment:
Was this just a bug before or has something changed such that now we don't see the '"' character?

Reply:
I'm fairly sure that this is basically the same issue as https://github.com/MicrosoftEdge/JsDbg/pull/21

Maybe @MSLaguana could comment better

Reply:
Yeah I think it is the same; probably from when I switched from parsing the gdb mi format to doing everything via python.